src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
+
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = (props) => {
+  const { comments } = props;
+
+  return (
+    <div className="CommentList">
```

Comment:
```suggestion
export const CommentList: React.FC<Props> = ({ comments }) => (
    <div className="CommentList">
```

It is only comments as props. So I believe this is shorter and more understandable.